node/libs/roles/src/validator/messages/leader_commit.rs

```diff
@@ -54,6 +54,26 @@ pub enum CommitQCVerifyError {
     BadSignature(#[source] anyhow::Error),
 }
 
+/// Error returned by `CommitQC::add()`.
+#[derive(thiserror::Error, Debug)]
+pub enum CommitQCAddError {
+    /// Inconsistent views.
+    #[error("Trying to add a message from a different view")]
+    InconsistentViews,
+    /// Signer not present in the committee.
+    #[error("Signer not in committee: {signer:?}")]
+    SignerNotInCommittee {
+        /// Signer of the message.
+        signer: Box<validator::PublicKey>,
```

Comment:
nit: key is not that large. Any specific reason to box it?

Reply:
like, it is fine as is, I'm just curious.

Reply:
Clippy was complaining about it. Not sure why as its not complaining in other similar parts of the code